src/ArrayLayouts.jl

```diff
@@ -52,6 +52,14 @@ else
     const CNoPivot = NoPivot
 end
 
+if VERSION ≥ v"1.10-"
```

Comment:
I don't think this version is correct; AFAICT it should be
```suggestion
if VERSION ≥ v"1.11.0-DEV.21"
```

Reply:
Btw, in case you weren't aware, given a git commit SHA on the Julia repo, you can use `contrib/commit-name.sh <sha>` to get the corresponding version number. For reference, the commit associated with merging the linked PR is `feb2988b3a21968410267378b910ce67726a51d8`.

Reply:
Actually, I'd love to backport the corresponding commit. But if that's too hot, then I'll update the version number.

Reply:
I don't think it can be backported given that it isn't a bugfix. The usual process is that a release series becomes frozen for features and minor changes once its corresponding `release-1.x` branch is created. (This is, of course, putting aside the scale of breakage observed in the ecosystem after the PR was merged.)

Reply:
I understand, though I find it sad, because it was sitting there for so long.

Reply:
I realized that the bound doesn't need to be sharp for that commit. This is just about the definition of the alias, which we have already in v1.9, but without the two type parameters. Starting from v1.10 we have the two type parameters included, and only from that specific v1.11-commit onwards `AbstractTriangular` only has a single type parameter. So, perhaps it's ok to set 1.10 again so that we can clean this little piece of code up once we lift the lower bound to the new (expected) LTS?